From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ahmed ARIF <arif.ing@outlook.com>
Date: Mon, 12 Jan 2026 00:00:00 +0000
Subject: [PATCH] mingw-w64: add truncl support for AArch64

Use trunc() for truncl on AArch64 where long double may have different
representation.

--- a/mingw-w64-crt/math/truncl.c
+++ b/mingw-w64-crt/math/truncl.c
@@ -11,6 +11,8 @@
 {
 #if __SIZEOF_LONG_DOUBLE__ == __SIZEOF_DOUBLE__
   return trunc(_x);
+#elif defined(__aarch64__)
+  return trunc(_x);
 #else
   long double retval = 0.0L;
   unsigned short saved_cw;
